src/main/java/io/debezium/connector/vitess/connection/ColumnMetaData.java

```diff
@@ -17,14 +17,19 @@
 public class ColumnMetaData {
     private final String columnName;
     private final VitessType vitessType;
-    private final boolean key;
     private final boolean optional;
+    // key and uniqueKey cannot both be true as the same time.
```

Comment:
// key and uniqueKey. Cannot both be true as the same time.

Reply:
fixed